### DISCUSSION TITLE
Transition document "LedgerSMB for SQL Ledger users"

### DISCUSSION THREAD

**Discussion by ehuelsmann** [ledgersmb/LedgerSMB#6642] [Ideas]
Subject says it all

**Comment by sbts** [comment]
@ylavoie Can you draw on your recent experience to start writing this document?
You certainly know the SL => LSMB migration tools better than the rest of us at this point

Also bumping the milestone to 1.6 so it doesn't get lost, although we really should try and get it started during the 1.5 lifecycle
